src/index.tsx

```diff
@@ -0,0 +1,63 @@
+import React, {
+  forwardRef,
+  ReactNode,
+  FunctionComponent,
+  createContext,
+  HTMLAttributes,
+  DetailedHTMLProps,
+  createElement,
+} from 'react'
+import PropTypes from 'prop-types'
```

Comment:
As above, not sure if we should require that the end-user have `prop-types` installed